dotCMS/src/main/java/com/dotmarketing/portlets/categories/business/CategoryFactoryImpl.java

```diff
@@ -721,7 +721,13 @@ public void sortChildren(String inode)  throws DotDataException {
 			statement = conn.createStatement();
 			String sql;
 
-            PreparedStatement createSortPreparedStatement = conn.prepareStatement( catSQL.getCreateSortChildren() );
+            if ( DbConnectionFactory.isOracle() ){
+                //For Oracle we need to avoid ORA-01027 by creating the table before.
+                sql = catSQL.createCategoryReorderTable();

```

Comment:
Oscar, will this throw an error if the table is already created?  Do we need a check to see if the table already exists?